fast_randint.h

```diff
@@ -0,0 +1,6 @@
+#ifndef FAST_RANDINT_H
+#define FAST_RANDINT_H
+
+int fast_randint(int max);  // Returns a random integer 0..(max-1)
```

Comment:
Put in a comment that `max` must be at most 52.

Reply:
Should these be ints? They're stored as chars, but used as indices into arrays. Unclear what the right type is.